### ISSUE TITLE
Creating table with Bool column creates a wrong type.

### ISSUE THREAD

**Issue by reygoch** [valderman/selda#53]
I have a following table definition :

```
type UserRow = RowID :*: Text :*: Text :*: Bool :*: UTCTime :*: UTCTime

user :: Table UserRow
user = table "User"
  $   autoPrimary "id"

  :*: required "username"
  :*: required "password"

  :*: required "admin"

  :*: required "dateCreated"
  :*: required "dateModified"
```

Running `tryCreateTable user` creates a new table in database but with `admin` columns data type set to `integer`.

And when I try to insert new user I get following error :

```
*** Exception: SqlError "error executing query `INSERT INTO "User" ("id", "username", "password", "admin", "dateCreated", "dateModified") VALUES (DEFAULT, $1, $2, $3, $4, $5)':

ERROR:  column "admin" is of type integer but expression is of type boolean

LINE 1: ...reated", "dateModified") VALUES (DEFAULT, $1, $2, $3, $4, $5...
HINT:  You will need to rewrite or cast the expression."
```

**Comment by valderman** [comment]
Fixed as of 0.1.11.2. Thanks for the catch!

**Comment by reygoch** [comment]
damn, I was just about to issue a pull request :)
